src/Sylius/Behat/Context/Ui/Admin/ManagingLocalesContext.php

```diff
@@ -187,4 +188,12 @@ public function thisLocaleShouldBeDisabled(LocaleInterface $locale)
             sprintf('Locale %s should be disabled but it is not', $locale->getCode())
         );
     }
+
+    /**
+     * @Then I should not be able to choose :name
+     */
+    public function iShouldNotBeAbleToChoose($name)
+    {
+        expect($this->createPage)->toThrow(ElementNotFoundException::class)->during('chooseName', [$name]);
```

Comment:
We agreed to replace expect and catching exception with Assert and dedicated method on page that checks ``isSelectOptionAvailable`` or smth similar.